src/main/java/de/siphalor/amecs/api/KeyBindingUtils.java

```diff
@@ -56,12 +61,40 @@ public static void setLastScrollAmount(float lastScrollAmount) {
 		return idToKeyBindingMap;
 	}
 
+	/**
+	 * Unregisters a keybinding from input querying but is NOT removed from the controls GUI 
+	 * <br> if you unregister a keybinding which is already in the controls GUI you can call {@link #registerHiddenKeyBinding(KeyBinding)} with this keybinding to undo this
+	 * <p> This is possible even after the game initialized
+	 * 
+	 * @param keyBinding 
+	 * @return whether the keyBinding was removed. It is not removed if it was not contained
+	 */
+	public static boolean unregisterKeyBinding(KeyBinding keyBinding) {
+		return KeyBindingManager.unregister(keyBinding);
+	}
+	
```

Comment:
> If this function is really functionally different to unregisterKeyBinding then it should have a different name.

What `unregisterKeyBinding` do you mean? The method in the same class with the translationKey as its argument. Or do you mean the method of the fabric api?

Because the two methods in this class called `unregisterKeyBinding` do the excat same thing. Only that the method with the translationKey as argument first gets the corresponding keybinding.

Reply:
> Not really sure where this would come to use.

I have a use case for this in my own mod. That was the reason to expose it as API. Otherwise I would need to reflect oder mixin into it.
The use case: I want to listen for a keybinding press (therefore it must be registered). But I have my own custom GUI in which the keybindings are shown (and configureable). And therefore it should not appear in the normal controls gui.

Reply:
Ah, I see the use case.

The other method calls `getIdToKeyBindingMap().remove()` though which the other one doesn't. Just a minor difference.

Reply:
> The other method calls getIdToKeyBindingMap().remove() though which the other one doesn't. Just a minor difference.
Yes you are right. I think the method with the keybinding as argument should call the other one. Then both will remove the keybinding from both lists